activiti-cloud-query-service/activiti-cloud-starter-query/src/main/resources/config/query/liquibase/changelog/03-alter.oracle.schema.7.1.0-m9.sql

```diff
@@ -0,0 +1,6 @@
+alter table process_instance
```

Comment:
The suffix of the file name should be `m10` instead of `m9`. As it will go in `m10` release.